src/styles/page-template/_template.njk

```diff
@@ -14,7 +14,9 @@
 
 {% if pageConfig.cdn or release_version %}
     {# Production #}
-    {% set assetsURL = "https://cdn.ons.gov.uk/sdc/design-system/" + (pageConfig.cdn or release_version) %}
+    {% set cdn_url = pageConfig.cdn.url or "https://cdn.ons.gov.uk/sdc/design-system" %}
+    {% set trailing_slash = "/" if cdn_url | last != "/" else "" %}
+    {% set assetsURL = cdn_url + trailing_slash + (pageConfig.cdn.version or release_version) %}
```

Comment:
Minor, the `/` isn't trailing as we're using it between two strings, and I think flipping the logic to be positive makes it clearer.
```suggestion
    {% set slash = "" if cdn_url | last == "/" else "/" %}
    {% set assetsURL = cdn_url + slash + (pageConfig.cdn.version or release_version) %}
```